tests/test_builder.py

```diff
@@ -4,7 +4,7 @@
 
 @pytest.fixture(scope="module")
 def rootdir():
-    return path(__file__).parent.abspath() / "roots/builder"
+    return path(__file__).parent.abspath().joinpath("roots").joinpath("builder")
```

Comment:
```suggestion
    return path(__file__).parent.abspath() / "roots" / "builder"
```
^ Does that work? It reads better.

Reply:
Does the forward slash create an actual forward slash, or use the os.sep(). Joinp path uses is.sep()

Reply:
`/` just calls joinpath.